Server/Server.Core/Server.Core.Static/Settings.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <map>
+
+namespace Server::Builder
+{
+struct ParamType
+{
+    constexpr static char const* ServerPort  = "--serverport";
+    constexpr static char const* HostAddress = "--hostaddr";
+    constexpr static char const* DBName      = "--dbname";
+    constexpr static char const* DBPort      = "--port";
+    constexpr static char const* DBUser      = "--user";
+    constexpr static char const* DBPassword  = "--password";
+};
+
+class Settings
+{
+public:
+    Settings() = default;
+
+    inline Settings& SetValue(std::pair<std::string, std::string> value)
+    {
+        _settings.try_emplace(value.first, value.second);
+        return *this;
+    }
+
+    std::string GetValue(const std::string& key) const
+    {
+        if (_settings.find(key) != _settings.end())
```

Comment:
if (auto it = _settings.find(key); it != _settings.end()) //since c++17 https://en.cppreference.com/w/cpp/language/if
    return *it;

Reply:
It's a cool comment. I'll do so.
Thank you.